db/migrate/helpers/delete-content-item.rb

```diff
@@ -0,0 +1,21 @@
+module Helpers
+  module DeleteContentItem
+    def self.destroy_supporting_objects(content_items)
```

Comment:
Good point about the readme

hmm the `Array` would be unnecessary as ActiveRecord handles it being a model or an array. Do you mean to add that so it's explicit that it accepts a single item or a collection?